### DISCUSSION TITLE
Accessing HLS imagery on Azure Blob Storage

### DISCUSSION THREAD

**Discussion by Dputnam-VT** [microsoft/PlanetaryComputer#306] [Q&A]
Hello! I'm new to the planetary computer, and my research would greatly benefit from being able to work with Harmonized Landsat Sentinel-2 data with cloud computing. I tried running the example notebook [here](https://nbviewer.org/github/microsoft/AIforEarthDataSets/blob/main/data/hls.ipynb) but I'm getting a HTTPS error code 403 when running on the JupyterHub. 

Not sure what I'm doing wrong, there's something in the doc about a SAS token but it doesn't describe how it's supposed to be used when making requests.

Thanks for the help!

**Comment by Kongstad** [comment]
I am also looking into using HLS for a project and I could access the notebook yesterday, but now I run into a 503 error. 

```
Error 503 Backend is unhealthy
Backend is unhealthy

Error 54113
```

Are there any plans in the forseeable future to include HLS in the same manner as Modis/Sentinel-2 etc. in your systems?

**Comment by TomAugspurger** [comment]
I think nbviewer is having some issues today. The notebooks are also available at https://github.com/microsoft/AIforEarthDataSets/.

I think that our HLS is quite old (IIRC the version has been superseded by a newer version, and I think it might not have the latest data).

We don't currently have plans to update that pipeline and get STAC metadata for it.

________________________________

**Reply by Kongstad** [reply]
Thank you for your answer Tom. 
I will try the nbviewer later on. All links to the notebooks seems to be dead on my end. 
So if I were to implement HLS, it would make more sense to go directly to the source over at [NASA/USGS](https://lpdaac.usgs.gov/data/get-started-data/collection-overview/missions/harmonized-landsat-sentinel-2-hls-overview/) ?

**Comment by TomAugspurger** [comment]
Possibly. I'm not sure what the best way to access it is.
________________________________
